### PR TITLE
Adjust partitioning_raid for new shortcuts

### PR DESCRIPTION
**Pull request by Jiawei517**
Add version check for SLES15
Add needle check for expert partitioner drop down menu
Select expert partitioner option for version latest then 15

- see POO#27436

- Related ticket: https://progress.opensuse.org/issues/27436
- Needles: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/565
- Verification run: http://10.67.17.147/tests/1709#step/partitioning_raid/1

### CODE REVIEW

**Comment by okurz** [comment]
The needle PR mentioned is wrong

**Comment by Jiawei517** [comment]
Updated needle MR link.

File: tests/installation/partitioning_raid.pm [hunk 1]
```diff
@@ -159,6 +159,11 @@ sub run {
     # create partitioning
     send_key(is_storage_ng() ? $cmd{expertpartitioner} : $cmd{createpartsetup});
 
+    if (sle_version_at_least '15') {
```
Comment (rwx788) [inline tests/installation/partitioning_raid.pm]: This sub menu is not yet in leap, but it makes sense for storage_ng only, so we should put it in next if (is_storage_ng) block and additionally check if it's sle using is_sle method.

File: tests/installation/partitioning_raid.pm [hunk 2]
```diff
@@ -159,6 +159,11 @@ sub run {
...
+        assert_screen 'expertpartitioner_list';
+        send_key 'tab';
+        send_key 'ret';
```
Comment (rwx788) [inline tests/installation/partitioning_raid.pm]: We had bad experience when used send_keys without verifying that next window is there, let's wrap it in wait_screen_change at least, or we may end up in situation when we press next button before it is there.

**Comment by Jiawei517** [comment]
Closed this one, it is duplicated with PR#3899.

**Comment by Jiawei517** [comment]
Closed this one, it is duplicated with PR#3899.

**Pull request closed without merging**